src/benchmark_runner.cc

```diff
@@ -80,6 +80,10 @@ BenchmarkReporter::Run CreateRunReport(
   BenchmarkReporter::Run report;
 
   report.run_name = b.name();
+
+  if (FLAGS_benchmark_dry_run) {
```

Comment:
i don't think we need to add this to every benchmark.  i think we can add it to the context.

Reply:
Ok, ill change that!